README.md

```diff
@@ -1,3 +1,5 @@
+[![Coverage Status](https://coveralls.io/repos/github/danbilokha/covid19liveupdates/badge.svg?branch=master)](https://coveralls.io/github/danbilokha/covid19liveupdates?branch=master)
```

Comment:
Can't wait to see this

Reply:
34% 👍 Even though we're testing only 1 file

Reply:
Thant's because it hasn't been mocked, I suppose